package.json

```diff
@@ -30,7 +30,8 @@
     "babel-preset-react": "^6.24.1",
     "express": "^4.16.1",
     "flow-bin": "^0.41.0",
-    "jest": "^21.2.1",
+    "jest": "23",
```

Comment:
**question:** Only 23? Why not the latest?

Reply:
Oh, I see you address that in the PR description. 👍🏻

Reply:
I ran into issues upgrading to the latest.  I'll try again and post the errors.

Reply:
The error I was running into was the following:
```
 FAIL  __tests__/test.js
  ● Test suite failed to run

    Plugin/Preset files are not allowed to export objects, only functions. In /Users/kevinbarabash/khan/react-loadable/node_modules/babel-preset-es2015/lib/index.js

      at createDescriptor (node_modules/@babel/core/src/config/config-descriptors.ts:350:11)
          at createDescriptor.next (<anonymous>)
      at evaluateSync (node_modules/gensync/index.js:251:28)
      at node_modules/gensync/index.js:31:34
          at Array.map (<anonymous>)
      at Function.sync (node_modules/gensync/index.js:31:22)
      at Function.all (node_modules/gensync/index.js:210:24)
      at createDescriptors (node_modules/@babel/core/src/config/config-descriptors.ts:261:38)
          at createDescriptors.next (<anonymous>)
```